javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,51 @@
+var width = 30;
+var height = 40;
+var area = width * height;
+
+console.log('Value of area: ', area);
+console.log('type of area: ', typeof area);
+
+var bill = 23;
+var payment = 40;
+var change = payment - bill;
+
+console.log('Value of change: ', change);
+console.log('type of change: ', typeof change);
+
+var quizzes = 5;
+var midterm = 3;
+var final = 1;
+var grade = (quizzes + midterm + final) / 3;
+
+console.log('Value of grad: ', grade);
+console.log('type of grade: ', typeof grade);
+
+var firstName = 'Euichang';
+var lastName = 'Park';
+var fullName = firstName + ' ' + lastName;
+
+console.log('value of fullName: ', fullName);
+console.log('type of fullName: ', typeof fullName);
+
+var pH = 6;
+var isAcidic = false;
+if (pH < 7) {
```

Comment:
you don't want any `if` statements in here, you are just saving the result of the expression into a variable and then logging it out.